benchmark/pull/server.yml.example

```diff
@@ -0,0 +1,11 @@
+---
+# log-level: debug
+scrape-configs:
+  - job-name: testing
+    enabled-profiles: [cpu, mem]
+    static-configs:
+      - application: pull-target
+        targets:
+          - pull-target:4042
+        labels:
+          pod: pod-0
```

Comment:
I think we could even create a fake discovery mechanism that would produce targets (with unique scrape url).

Reply:
Yeah, that would be useful here, although generating these configs a is also pretty easy.

What do you mean by unique scraping URL? Is it important for them to have different ones?